HISTORY.rst

```diff
@@ -3,6 +3,38 @@
 History
 =======
 
+
+1.0.0 (2020-10-XX)
+------------------
+
+Breaking changes:
+
+* Drop support for Python 2.7 and pypy (see #727, #744, #748, #749, #754, #755, #758, #761, #763, #764, #777, #783)
+* Now ``dateparser.parse()`` is deterministic and don't try previous locales. On the other hand, ``DateDataParser.get_date_data()`` don't try the previous locales by default (see #781)
```

Comment:
```suggestion
* Now ``dateparser.parse()`` is deterministic and doesn't try previous locales. On the other hand, ``DateDataParser.get_date_data()`` doesn't try the previous locales by default (see #781)
```

Reply:
From reading this, I’m a bit confused because it says “on the other hand” but seems to state that `DateDataParser.get_date_data()` behaves the same as `parse` (by default).

Reply:
@Gallaecio 

"[...] is deterministic and doesn't try previous locales, while ``DateDataParser.get_date_data()`` doesn't  try the previous locales by default"

would sound better this?

I'm not sure how to write this... What I want to say is:
* `dateparser.parse()` is now deterministic and it doesn't try the previously used locales.
* `DateDataParser.get_date_data()` by default doesn't try the previously used locales.

Could you help me to express it better? :sweat_smile:

Reply:
Maybe just “also” instead of “in the other hand” would make things more clear to me. Let me look again into the actual changes and get back to you.